src/legacy/server/config/schema.js

```diff
@@ -231,6 +231,15 @@ export default () =>
       locale: Joi.string().default('en'),
     }).default(),
 
+    // temporarily moved here from the (now deleted) kibana legacy plugin
+    kibana: Joi.object({
+      enabled: Joi.boolean().default(true),
+      index: Joi.string().default('.kibana'),
+      autocompleteTerminateAfter: Joi.number().integer().min(1).default(100000),
+      // TODO Also allow units here like in elasticsearch config once this is moved to the new platform
```

Comment:
which will be owning them? Both `input_control_vis` and `data` consume them

Reply:
Yea, cleaning up the legacy server's config may require some discussions. Fortunately this is not required to shut down the legacy plugin system